extras/userdoc/md/documentation/part-4-topologically-structured-networks.md

```diff
@@ -335,9 +351,4 @@ listed in NTUM Section 4.1, are:
 ## References
 
 \[1\]<a id="1"></a>  Hans Ekkehard Plesser and Håkon Enger NEST Topology User
- Manual
-
-## Acknowledgments
```

Comment:
No, I removed this on purpose. Many people have contributed to this documentation now (although it's still mostly me ;) ), there is no particular reason to single out Sarah.